tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -360,14 +360,16 @@ public function adminDeletesUserUsingTheGraphApi(string $user, ?string $byUser =
 	/**
 	 * remove user from group
 	 *
-	 * @param string $groupId
-	 * @param string $userId
+	 * @param string $group
+	 * @param string $user
 	 * @param string|null $byUser
 	 *
 	 * @return ResponseInterface
 	 * @throws GuzzleException
 	 */
-	public function removeUserFromGroup(string $groupId, string $userId, ?string $byUser = null): ResponseInterface {
+	public function removeUserFromGroup(string $group, string $user, ?string $byUser = null): ResponseInterface {
+		$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id") ?: WebDavHelper::generateUUIDv4();
```

Comment:
```suggestion
		$groupId = $this->featureContext->getAttributeOfCreatedGroup($group, "id") ?? WebDavHelper::generateUUIDv4();
```

Reply:
`$this->featureContext->getAttributeOfCreatedGroup($group, "id")` generate false when group is given as `empty string`  so, ?: is used

Reply:
`?:` is the short form of ternary operator in php